### PR TITLE
More oovpa updates

### PR DESCRIPTION
**Pull request by RadWolfie**
TLDR section
---

There was numerous of OV pairs are not in a group or at least have beginning of the offset. Sadly, this is not a full complete set of DSOUND reverse engineered symbols. Yet, is passing the XbSymbolUnitTest with exception of XFileCreateMediaObject and XWaveFileCreateMediaObject symbols in 3936 and 4134, I think.

**NOTE:** This pull request is currently low risk since XbSymbolUnitTest is passing almost all of build versions I have at the moment. There are several build versions I do not have currently.

I had compiled a cache log between Cxbx-Reloaded's master vs generated cache tool with this pull request branch. Below are what's been added, including couple corrections and renamed.

---
Comparison between master against this pull request
---

3925,3936:
* CDirectSoundBufferSettings_SetBufferData
* CDirectSoundBuffer_SetAllParameters
* CDirectSoundBuffer_SetConeAngles
* CDirectSoundBuffer_SetEG
* CDirectSoundBuffer_SetFilter
* CDirectSoundBuffer_SetFrequency
* CDirectSoundBuffer_SetHeadroom
* CDirectSoundBuffer_SetI3DL2Source
* CDirectSoundBuffer_SetLFO
* CDirectSoundBuffer_SetMixBinVolumes
* CDirectSoundBuffer_SetMixBins
* CDirectSoundBuffer_SetMode
* CDirectSoundBuffer_SetPitch
* CDirectSoundBuffer_SetVolume
* CDirectSoundStream_Constructor
* CMcpxBuffer_SetBufferData

4039:
* CDirectSoundBufferSettings_SetBufferData
* CDirectSoundStream_Constructor
* CDirectSoundVoice_CommitDeferredSettings
* CHRTFSource_SetLightHRTF5Channel
* CLightHRTFSource_GetCenterVolume
* CMcpxAPU_ServiceDeferredCommandsLow
* CMcpxAPU_SetMixBinHeadroom
* CMcpxBuffer_SetBufferData
* CMcpxVoiceClient_Commit3dSettings
* CMcpxVoiceClient_SetPitch
* DirectSoundOverrideSpeakerConfig
* DirectSoundUseLightHRTF
* XAudioCreateAdpcmFormat (renamed)
* XAudioDownloadEffectsImage
* CMcpxStream_Stop (????)
* CMcpxStream_Stop_Ex (may not exist)

4242:
* CDirectSoundBufferSettings_SetBufferData
* CDirectSoundStream_Constructor
* CDirectSoundVoice_CommitDeferredSettings
* CDirectSound_DoWork
* CMcpxAPU_ServiceDeferredCommandsLow
* CMcpxAPU_SetMixBinHeadroom
* CMcpxBuffer_SetBufferData (corrected)
* CMcpxStream_Stop
* CMcpxStream_Stop_Ex
* CMcpxVoiceClient_Commit3dSettings
* CMcpxVoiceClient_SetPitch
* DirectSoundOverrideSpeakerConfig
* XAudioCreateAdpcmFormat (renamed)
* XFileCreateMediaObject
* XFileCreateMediaObjectEx
* XWaveFileCreateMediaObject
* XWaveFileCreateMediaObjectEx

---

4361 and later are mostly the same new additions.

4361, 4432, 4721, 4831, 4928, 5028, 5120, 5233, 5344, 5455, 5558:
* CDirectSoundVoice_CommitDeferredSettings

4361, 4432, 4721, 4831, 4928, 5028, 5120, 5233, 5344, 5455, 5558, 5788, 5849:
* CDirectSoundBufferSettings_SetBufferData
* CDirectSoundStream_Constructor
* CDirectSoundVoice_CommitDeferredSettings
* CDirectSound_DoWork
* CMcpxAPU_ServiceDeferredCommandsLow
* CMcpxAPU_SetMixBinHeadroom
* CMcpxBuffer_SetBufferData (corrected)
* CMcpxStream_Stop
* CMcpxStream_Stop_Ex
* CMcpxVoiceClient_Commit3dSettings
* CMcpxVoiceClient_SetPitch
* DirectSoundOverrideSpeakerConfig
* XAudioCreateAdpcmFormat (renamed)

5120, 5233:
* XAudioSetEffectData

5233, 5344, 5455, 5558, 5788, 5849:
* CLightHRTFSource_GetCenterVolume

5455, 5558, 5788, 5849:
* XAudioCalculatePitch

### CODE REVIEW

**Comment by RadWolfie** [comment]
To be clear, not every OOVPAs had been thorough and I will fix any weak OOVPAs in the future.

Plus I am planning on re-update all OOVPA databases for couple tasks (per pull request).
